woocommerce/woocommerce-ios
# Polish domain purchase success UI

Part of #8558 


## Description


Previously, the domain purchase success view was implemented with a barebone design that just shows the purchased domain name, title, and CTA to enable the integration first. In this PR, the view was updated to match design.

## Testing instructions


Prerequisite: a WC store **with a WPCOM plan** and **without a domain credit** is required, make sure the proxy is disabled so that you can see the web checkout page.

- Log in if needed, and continue with a WPCOM store
- Go to the Menu tab
- Tap on the settings CTA
- Tap the `Domain` row
- Tap `Add a domain` or `Search for a domain` depending on if there are existing domains --> domain selector with paid domains should be shown, and each domain should have price info (with the barebone UI)
- Feel free to search for other domains
- Select a domain and tap `Continue`
- Complete the web checkout --> when the checkout is successful, it should navigate to the updated success screen with the purchased domain
- Tap `Done` --> it should navigate back to the domain settings, and the new domain should be shown shortly

## Screenshots


color schemes | large title with a long domain name
-- | --
<img width="633" alt="Screenshot 2023-02-01 at 9 26 55 AM" src="https://user-images.githubusercontent.com/1945542/215923567-65d5ebd9-7ec3-4712-b4e5-9f84f077b4fd.png"> | <img width="296" alt="Screenshot 2023-02-01 at 9 30 12 AM" src="https://user-images.githubusercontent.com/1945542/215923577-26ff071b-69ac-41b9-909b-e334fb99c827.png">


---
